src/kernels/variables/types.ts

```diff
@@ -27,6 +27,11 @@ export interface IJupyterVariable {
     fileName?: string;
 }
 
+export const IRootDirectory = Symbol('IRootDirectory');
```

Comment:
I don't think this is necessary. We already have IExtensionContext, which has an extensionUri property on it. That would be the root directory.

I also think IRootDirectory is a little ambiguous. Root of what? Could be workspace. Could be vscode. Could be extension.